lib/rspec/rails/configuration.rb

```diff
@@ -71,6 +71,9 @@ def self.initialize_configuration(config) # rubocop:disable Metrics/MethodLength
       config.add_setting :fixture_path
       config.include RSpec::Rails::FixtureSupport, :use_fixtures
 
+      # Prevent test from failing on undefined method used in ActiveSupport testing helpers
+      config.include ActiveSupport::Testing::Assertions
```

Comment:
Do we need to check if this is defined? / Can we pull it into one of our existing modules?

Reply:
I was very curious about the CI. 
I am pretty confident. This class exist since at least Rails 4.2 https://github.com/rails/rails/blob/4-2-stable/activesupport/lib/active_support/testing/assertions.rb  

I tried in `RailsExampleGroup` but it was not working, I still had the same error. I will keep digging.

Reply:
Ok! I'll wait for your experiments to finish :)

Reply:
I added a check on version. It's pretty basic but I think enough for our needs.